features/administration.feature

```diff
@@ -48,12 +51,13 @@ Feature: Bookstore Administration Panel
   Scenario: Editing a book
     Given I am logged into the admin panel
     And I am logged into the site
-    And there is a book named "Book To Be Edited" valued at "$135.99"
+    And there is a book named "Book To Be Edited" 
+    And the price of the book is "$135.99"
     When I visit the admin books url
-    And I click "Edit" for the book "Book To Be Edited"
+    And I edit the book with the title "Book To Be Edited"
     And I change the book name to "Book That Has Been Edited"
     And I change the book price to "222.22"
-    And I click "Submit"
+    And I click the "Update Book" button
     And I visit the admin books url
     Then I see the book "Book That Has Been Edited"
     When I visit the public book index
```

Comment:
The button is an Update Book button, not submit, so I changed it to match what the button says.  Should I change the button to Submit, or is there a test for just standard submit buttons?  Thanks